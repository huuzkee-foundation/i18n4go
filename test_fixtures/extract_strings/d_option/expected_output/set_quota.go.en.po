# filename: d_option/input_files/org/set_quota.go, offset: 793, line: 30, column: 16
msgid "set-quota"
msgstr "set-quota"

# filename: d_option/input_files/org/set_quota.go, offset: 821, line: 31, column: 16
msgid "Assign a quota to an org"
msgstr "Assign a quota to an org"

# filename: d_option/input_files/org/set_quota.go, offset: 858, line: 32, column: 10
msgid "CF_NAME set-quota ORG QUOTA\n\n"
msgstr "CF_NAME set-quota ORG QUOTA\n\n"

# filename: d_option/input_files/org/set_quota.go, offset: 897, line: 33, column: 4
msgid "TIP:\n"
msgstr "TIP:\n"

# filename: d_option/input_files/org/set_quota.go, offset: 911, line: 34, column: 4
msgid "   View allowable quotas with 'CF_NAME quotas'"
msgstr "   View allowable quotas with 'CF_NAME quotas'"

# filename: d_option/input_files/org/set_quota.go, offset: 1153, line: 40, column: 20
msgid "Incorrect Usage"
msgstr "Incorrect Usage"

# filename: d_option/input_files/org/set_quota.go, offset: 1633, line: 64, column: 13
msgid "Setting quota %s to org %s as %s..."
msgstr "Setting quota %s to org %s as %s..."

