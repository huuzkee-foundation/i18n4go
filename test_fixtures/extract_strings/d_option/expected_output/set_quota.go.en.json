[{"id":"set-quota","translation":"set-quota"},{"id":"Assign a quota to an org","translation":"Assign a quota to an org"},{"id":"CF_NAME set-quota ORG QUOTA\n\n","translation":"CF_NAME set-quota ORG QUOTA\n\n"},{"id":"TIP:\n","translation":"TIP:\n"},{"id":"   View allowable quotas with 'CF_NAME quotas'","translation":"   View allowable quotas with 'CF_NAME quotas'"},{"id":"Incorrect Usage","translation":"Incorrect Usage"},{"id":"Setting quota %s to org %s as %s...","translation":"Setting quota %s to org %s as %s..."}]
